src/components/DropdownList/DropdownList.tsx

```diff
@@ -0,0 +1,33 @@
+import { Person } from '../../types/Person';
+import './DropdownList.scss';
+
+type Props = {
+  peoples: Person[] | null;
+  onSelect: (person: Person) => void;
+};
+
+export const DropdownList: React.FC<Props> = ({
+  peoples,
+  onSelect,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        { peoples
+          ? peoples.map((man, index) => (
+            <div
+              key={`${man.name}-${man.born}`}
+              className="dropdown-item"
+              onClick={() => onSelect(man)}
+              onKeyDown={() => onSelect(man)}
+              role="button"
```

Comment:
It's not necessary to set role="button" on div if you are making it clickable. To improve accessibility, consider using a button instead of a div for clickable items.